Gruntfile.js

```diff
@@ -53,7 +53,8 @@ module.exports = function (grunt) {
                             '!binary-com-jquery-dialogextended/**', 'binary-com-jquery-dialogextended/jquery.dialogextend.min.js',
                             'binary-com-jquery-ui-timepicker/jquery.ui.timepicker.js', 'binary-com-jquery-ui-timepicker/jquery.ui.timepicker.css',
                             'text/text.js',
-                            'regenerator-runtime/*'
+                            'regenerator-runtime/*',
+                            'chosen-js/*'
```

Comment:
@4p00rv I don't see this dependency in package.json. Where is it? And any specific reason why we need this and cannot use Jquery-UI selectbox?